jupyterhub/orm.py

```diff
@@ -54,6 +56,38 @@ def process_result_value(self, value, dialect):
         return value
 
 
+class OptionalEncrypted(EncryptedType):
+    def __init__(self, type_in=None, key=None, engine=None, **kwargs):
+        try:
+            import cryptography
```

Comment:
I think setting auth_state should probably raise rather than warn in the absence of encryption